python/ray/autoscaler/v2/scheduler.py

```diff
@@ -187,6 +190,7 @@ def __init__(
         labels: Dict[str, str],
         status: SchedulingNodeStatus,
         im_instance_id: str = "",
+        im_instance_status: str = None,
```

Comment:
btw, is it actually a string? Maybe the type hint should be `Instance.InstanceStatus` instead.